packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -164,67 +169,101 @@ export class ZkEvmProvider implements Provider {
     return ethSigner;
   }
 
+  async #callSessionActivity() {
+    const sendTransactionClosure = async (params: Array<any>, flow: Flow) => {
+      const ethSigner = await this.#getSigner();
+      return await sendTransaction({
+        params,
+        ethSigner,
+        guardianClient: this.#guardianClient,
+        rpcProvider: this.#rpcProvider,
+        relayerClient: this.#relayerClient,
+        zkevmAddress: this.#zkEvmAddress!,
+        flow,
+      });
+    };
+    this.#eventEmitter.emit(ProviderEvent.ACCOUNTS_REQUESTED, {
+      environment: this.#config.baseConfig.environment,
+      sendTransaction: sendTransactionClosure,
+      walletAddress: this.#zkEvmAddress || '',
+      passportClient: this.#config.oidcConfiguration.clientId,
+    });
```

Comment:
Do we want dApp's to be able to subscribe to this event? If not, can I suggest that we use the `passportEventEmitter` (and move the event from `ProviderEvent.ACCOUNTS_REQUESTED` to `PassportEvents.ACCOUNTS_REQUESTED`?

The `eventEmitter` was intended to emit [EIP-1193 events](https://eips.ethereum.org/EIPS/eip-1193#events-1), where the `passportEventEmitter` was intended to emit internal events.

Reply:
Ahh, this is good insight, I wasn't aware. Thanks for this, I'll rework it to use the `passportEventEmitter` instead.